src/app/personside/infotabs/meldinger/traadvisning/verktoylinje/journalforing/JournalforSak.tsx

```diff
@@ -43,6 +44,8 @@ const SuksessStyling = styled.div`
 `;
 
 export function JournalforSak(props: Props) {
+    const dispatch = useDispatch();
+    const traaderSelector = useSelector((state: AppState) => state.restResources.tråderOgMeldinger);
```

Comment:
`traaderResource`..? også kan du bruke ´useRestResources´ for å spare noen tegn ;-)